space-cli/cmd/modules/ingress/generate.go

```diff
@@ -105,3 +105,49 @@ func generateIngressRouting() (*model.SpecObject, error) {
 
 	return v, nil
 }
+
+func generateIngressGlobal() (*model.SpecObject, error) {
+
+	project := ""
+	if err := input.Survey.AskOne(&survey.Input{Message: "Enter Project:"}, &project); err != nil {
+		return nil, err
+	}
+
+	headersKey := ""
+	if err := input.Survey.AskOne(&survey.Input{Message: "Enter Request Header Key:", Default: "Foo"}, &headersKey); err != nil {
+		return nil, err
+	}
+	headersValue := ""
+	if err := input.Survey.AskOne(&survey.Input{Message: "Enter Request Header Value:", Default: "Bar"}, &headersValue); err != nil {
+		return nil, err
+	}
+	headersOption := ""
+	if err := input.Survey.AskOne(&survey.Input{Message: "Enter Request Header Option:"}, &headersOption); err != nil {
```

Comment:
The `op` stands for operation and not option. I meant the user should get a select or something. Three possible values, set, add, del